src/main/kotlin/no/nb/mlt/wls/domain/ports/inbound/EditItem.kt

```diff
@@ -0,0 +1,48 @@
+package no.nb.mlt.wls.domain.ports.inbound
+
+import no.nb.mlt.wls.domain.model.Environment
+import no.nb.mlt.wls.domain.model.Item
+import no.nb.mlt.wls.domain.model.ItemCategory
+import no.nb.mlt.wls.domain.model.Packaging
+
+/**
+ * A port for editing items in the system.
+ *
+ * This port accepts an [Item] instance and new metadata which will replace the old metadata
+ * and returns an updated [Item]. It is designed to allow for the editing of items in the system, in cases
+ * where the item's metadata was wrong, or we got info about item after an order was made. The second use case
+ * exists because if we get an order with items we don't know, we will create a mostly empty [Item] with only
+ * the ID and host name set. This function lets catalogs update such items.
```

Comment:
This is ok. Can be made a bit more concise:
```suggestion
 * This port accepts an [Item] instance and new metadata which will replace the old metadata
 * and returns an updated [Item]. It is designed to allow for the editing of items in the system, in cases
 * where the item's metadata was wrong. The second use case
 * exists because if we get an order with items we don't know, we will create a mostly empty [Item] with only
 * the ID and host name set.
```